cmd/merge-sql-schemas/main.go

```diff
@@ -0,0 +1,101 @@
+package main
+
+import (
+	"database/sql"
+	"io/ioutil"
+	"log"
+	"os"
+	"path/filepath"
+	"regexp"
+	"sort"
+
+	_ "modernc.org/sqlite" // pure Go SQLite driver
+)
+
+func main() {
+	// Open an in-memory SQLite database.
+	db, err := sql.Open("sqlite", ":memory:")
+	if err != nil {
+		log.Fatalf("failed to open in-memory db: %v", err)
+	}
+	defer db.Close()
+
+	migrationDir := "tapdb/sqlc/migrations"
+	files, err := ioutil.ReadDir(migrationDir)
```

Comment:
`ioutil` shouldn't be used anymore. Use `os.ReadDir()` and `os.ReadFile` instead.